app/channel.go

```diff
@@ -464,14 +460,6 @@ func (a *App) GetGroupChannel(userIds []string) (*model.Channel, *model.AppError
 
 // UpdateChannel updates a given channel by its Id. It also publishes the CHANNEL_UPDATED event.
 func (a *App) UpdateChannel(channel *model.Channel) (*model.Channel, *model.AppError) {
-	userIds := strings.Split(channel.Name, "__")
```

Comment:
I would expect to see the `IsValid` here. This function is called from multiple places and is hard to track what are not validating if we remove it from here.

Reply:
Does it make sense to add it here when the store validates it anyway and the error is propagated up?

Reply:
Yes, that is a good point, I didn't realize that. Yes, if there is already validated at the store works for me. :)